app/src/main/java/com/pajato/android/gamechat/chat/fragment/CreateProtectedUsersFragment.java

```diff
@@ -0,0 +1,362 @@
+package com.pajato.android.gamechat.chat.fragment;
+
+import android.support.annotation.NonNull;
+import android.support.design.widget.TextInputEditText;
+import android.support.design.widget.TextInputLayout;
+import android.support.v4.app.FragmentActivity;
+import android.text.Editable;
+import android.text.TextUtils;
+import android.text.TextWatcher;
+import android.util.Log;
+import android.view.View;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.firebase.ui.auth.ui.TaskFailureLogger;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.EmailAuthProvider;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.ProviderQueryResult;
+import com.pajato.android.gamechat.R;
+import com.pajato.android.gamechat.chat.BaseChatFragment;
+import com.pajato.android.gamechat.common.DispatchManager;
+import com.pajato.android.gamechat.common.FabManager;
+import com.pajato.android.gamechat.common.ToolbarManager;
+import com.pajato.android.gamechat.credentials.Credentials;
+import com.pajato.android.gamechat.database.ProtectedUserManager;
+import com.pajato.android.gamechat.event.ClickEvent;
+
+import org.greenrobot.eventbus.Subscribe;
+
+import java.util.List;
+
+import static android.content.Context.INPUT_METHOD_SERVICE;
+import static com.pajato.android.gamechat.R.id.create_button_finish;
+import static com.pajato.android.gamechat.R.id.emailEditText;
+import static com.pajato.android.gamechat.R.id.email_next_button;
+import static com.pajato.android.gamechat.common.FragmentType.groupsForProtectedUser;
+import static com.pajato.android.gamechat.common.ToolbarManager.MenuItemType.helpAndFeedback;
+import static com.pajato.android.gamechat.common.ToolbarManager.MenuItemType.settings;
+
+/**
+ * Provide a fragment to create protected users.
+ */
+
+public class CreateProtectedUsersFragment extends BaseChatFragment {
+
+    // Private class constants.
+
+    /** The logcat tag. */
+    private static final String TAG = CreateProtectedUsersFragment.class.getSimpleName();
+
+    /** Firebase required password to be at least 6 characters */
+    private static final int PASSWORD_MIN = 6;
+
+    /** After testing email address, remember if it is new or already in the auth database */
+    private boolean accountIsKnown = false;
+
+    // Private inner classes
+
+    /** Handle email address provider check */
+    private class EmailOnSuccessListener implements OnSuccessListener<ProviderQueryResult> {
+        @Override
+        public void onSuccess(ProviderQueryResult result) {
+            RelativeLayout layout =
+                    (RelativeLayout) getActivity().
```

Comment:
Can you use mLayout rather than getActivity()?  If only to use 7 vs 13 characters.